src/components/proposals/proposal-view.vue

```diff
@@ -177,6 +192,7 @@ widget.proposal-view.q-mb-sm
     .column
       .text-h6.text-bold {{ title }}
       .text-italic.text-body {{ subtitle }}
+  //- version-history(:history="versionHistory") // TODO
```

Comment:
@evgenibir You need to write in TODO Also what needs to be done, so that other people know